courses/models/sectorsalary.py

```diff
@@ -26,12 +26,16 @@ def __init__(self, salary_data, display_language, institution_country_code):
 
             if institution_country_code == 'XF':
                 country_postfix = "_e"
+                self.country = "England"
```

Comment:
Do we need Welsh translations for country names?

Reply:
Yes we do